benchmarks/benchmarks/signal_filtering.py

```diff
@@ -102,3 +102,16 @@ def time_medfilt2d(self, threads):
 
     def peakmem_medfilt2d(self, threads):
         self._medfilt2d(threads)
+
+
+class FreqzRfft(Benchmark):
+    param_names = ['worN']
```

Comment:
You should also parametrize over `whole` and `nyquist` for completeness I think.